chromium_src/services/network/resolve_host_request.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_SINGAL_TEXT_RESULTS                     \
```

Comment:
don't we get these in `OnComplete` anyway which is called right after this?

Reply:
oh wait, I see, it calls response_client_->OnComplete directly when not pending, but then shouldn't we call `SignalNonAddressResults`  instead of duplicating here?

Reply:
SignalNonAddressResults calls class member **response_client_,**, but we call local instance of **response_client** it should be moved later [here](https://source.chromium.org/chromium/chromium/src/+/master:services/network/resolve_host_request.cc;l=69?q=resolve_host_request.cc&ss=chromium).
This is only for case when we got data from local dns cache, Chromium works with A/AAAA records and no need to call onTextResults. We are first here and maybe we should push this patch to upstream